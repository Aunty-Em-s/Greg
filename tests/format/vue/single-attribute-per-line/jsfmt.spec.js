run_spec(__dirname, ["vue"]);
run_spec(__dirname, ["vue"], { singleAttributePerLine: true });
