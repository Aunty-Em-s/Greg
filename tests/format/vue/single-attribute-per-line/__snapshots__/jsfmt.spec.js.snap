// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`single-attribute-per-line.vue - {"singleAttributePerLine":true} format 1`] = `
====================================options=====================================
parsers: ["vue"]
printWidth: 80
singleAttributePerLine: true
                                                                                | printWidth
=====================================input======================================
<template>
  <div data-a="1">
    Lorem ipsum dolor sit amet, consectetur adipiscing elit.
  </div>

  <div data-a="1" data-b="2" data-c="3">
    Lorem ipsum dolor sit amet, consectetur adipiscing elit.
  </div>

  <div data-a="Lorem ipsum dolor sit amet" data-b="Lorem ipsum dolor sit amet" data-c="Lorem ipsum dolor sit amet">
    Lorem ipsum dolor sit amet, consectetur adipiscing elit.
  </div>

  <div data-long-attribute-a="1" data-long-attribute-b="2" data-long-attribute-c="3">
    Lorem ipsum dolor sit amet, consectetur adipiscing elit.
  </div>

  <img src="/images/foo.png" />

  <img src="/images/foo.png" alt="bar" />

  <img src="/images/foo.png" alt="Lorem ipsum dolor sit amet, consectetur adipiscing elit." />
</template>

=====================================output=====================================
<template>
  <div data-a="1">Lorem ipsum dolor sit amet, consectetur adipiscing elit.</div>

  <div
    data-a="1"
    data-b="2"
    data-c="3"
  >
    Lorem ipsum dolor sit amet, consectetur adipiscing elit.
  </div>

  <div
    data-a="Lorem ipsum dolor sit amet"
    data-b="Lorem ipsum dolor sit amet"
    data-c="Lorem ipsum dolor sit amet"
  >
    Lorem ipsum dolor sit amet, consectetur adipiscing elit.
  </div>

  <div
    data-long-attribute-a="1"
    data-long-attribute-b="2"
    data-long-attribute-c="3"
  >
    Lorem ipsum dolor sit amet, consectetur adipiscing elit.
  </div>

  <img src="/images/foo.png" />

  <img
    src="/images/foo.png"
    alt="bar"
  />

  <img
    src="/images/foo.png"
    alt="Lorem ipsum dolor sit amet, consectetur adipiscing elit."
  />
</template>

================================================================================
`;

exports[`single-attribute-per-line.vue format 1`] = `
====================================options=====================================
parsers: ["vue"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
<template>
  <div data-a="1">
    Lorem ipsum dolor sit amet, consectetur adipiscing elit.
  </div>

  <div data-a="1" data-b="2" data-c="3">
    Lorem ipsum dolor sit amet, consectetur adipiscing elit.
  </div>

  <div data-a="Lorem ipsum dolor sit amet" data-b="Lorem ipsum dolor sit amet" data-c="Lorem ipsum dolor sit amet">
    Lorem ipsum dolor sit amet, consectetur adipiscing elit.
  </div>

  <div data-long-attribute-a="1" data-long-attribute-b="2" data-long-attribute-c="3">
    Lorem ipsum dolor sit amet, consectetur adipiscing elit.
  </div>

  <img src="/images/foo.png" />

  <img src="/images/foo.png" alt="bar" />

  <img src="/images/foo.png" alt="Lorem ipsum dolor sit amet, consectetur adipiscing elit." />
</template>

=====================================output=====================================
<template>
  <div data-a="1">Lorem ipsum dolor sit amet, consectetur adipiscing elit.</div>

  <div data-a="1" data-b="2" data-c="3">
    Lorem ipsum dolor sit amet, consectetur adipiscing elit.
  </div>

  <div
    data-a="Lorem ipsum dolor sit amet"
    data-b="Lorem ipsum dolor sit amet"
    data-c="Lorem ipsum dolor sit amet"
  >
    Lorem ipsum dolor sit amet, consectetur adipiscing elit.
  </div>

  <div
    data-long-attribute-a="1"
    data-long-attribute-b="2"
    data-long-attribute-c="3"
  >
    Lorem ipsum dolor sit amet, consectetur adipiscing elit.
  </div>

  <img src="/images/foo.png" />

  <img src="/images/foo.png" alt="bar" />

  <img
    src="/images/foo.png"
    alt="Lorem ipsum dolor sit amet, consectetur adipiscing elit."
  />
</template>

================================================================================
`;
